src/video_emotion/facial_recognition/facial_recognition.py

```diff
@@ -3,6 +3,8 @@
 import numpy as np
 import cv2
 
+from typing import Dict
```

Comment:
Typing should be all the way at the top as it is a base python module. Numpy and cv2 should be below as they are from pip.